src/components/Icon.jsx

```diff
@@ -1,5 +1,14 @@
+import React from "react";
+import Image from "next/image";
+
 const Icon = () => {
-  return <div>Icon</div>;
+  return (
+    <div className="container">
+      <div className="image">
+        <Image src="navbar-logo.svg" alt="Navbar-Logo" width={57} height={64} />
```

Comment:
import the image as an object instead of using width and height
i.e. import objectName from "/objectpath"
...
<Image src={objectName}